ocs_ci/utility/utils.py

```diff
@@ -1303,3 +1304,33 @@ def wait_for_co(operator):
     from ocs_ci.ocs.ocp import OCP
     ocp = OCP(kind='ClusterOperator')
     ocp.get(operator)
+
+
+def censor_values(data_to_censor, keys=None):
```

Comment:
why do we need "keys=None" parameter. we can this since we are using constants.config_keys_patterns_to_censor

Reply:
right, will do, sorry, missed it